projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/base-field/field-read.component.spec.ts

```diff
@@ -12,13 +12,15 @@ import { CaseFieldService } from '../../../services/case-fields/case-field.servi
 import { FieldTypeSanitiser } from '../../../services/form/field-type-sanitiser';
 import { FormErrorService } from '../../../services/form/form-error.service';
 import { FormValueService } from '../../../services/form/form-value.service';
+import { MockRpxTranslatePipe } from '../../../test/mock-rpx-translate.pipe';
 import { WizardPage } from '../../case-editor/domain/wizard-page.model';
 import { Wizard } from '../../case-editor/domain/wizard.model';
 import { PageValidationService } from '../../case-editor/services/page-validation.service';
 import { PaletteService } from '../palette.service';
 import { FieldReadComponent } from './field-read.component';
 import { PaletteContext } from './palette-context.enum';
 
+
```

Comment:
@DavidJayakumar - what's this extra blank line for?

Reply:
Removed extra line

Reply:
Done